rust/azure_iot_operations_connector/Cargo.toml

```diff
@@ -17,8 +17,12 @@ publish = true
 # azure_iot_operations_mqtt = { version = "0.9", path = "../azure_iot_operations_mqtt", registry = "aio-sdks"  }
 # derive_builder.workspace = true
 # log.workspace = true
-# tokio.workspace = true
+tokio.workspace = true
 # tokio-util.workspace = true
+notify = "5.1"
```

Comment:
notify and futures look like they're being used in other crates as well, so we can move them to have their version be defined in the workspace (like this: https://github.com/Azure/iot-operations-sdks/blob/main/rust/Cargo.toml#L28) and then update any Cargo.toml's that use them to do <crate name>.workspace = true. Ping me if you have any questions, it should be quick!